styles/_variables.scss

```diff
@@ -85,4 +85,11 @@ $spacing--desktop: 30px;
 
 $border--width: 1px;
 $border--radius: 8px;
+$focus--width: 5px;
 $font-zendesk-icons: "copenhagen-icons";
+
+/* HEADER SPECIFIC VARIABLES */
+$header__height--min: 80px;
```

Comment:
I recall you saying we were going to us `em`s why are these all `px` based?

Reply:
Ems don't work with the mixin I'm using.
I could probably refactor that.